src/App.js

```diff
@@ -2,6 +2,16 @@ import React from "react";
 import "./App.css";
 import { makeShuffledDeck } from "./utils.js";
 
+function RoundWinner(props) {
+  if (props.p1RoundsWon > props.p2RoundsWon) {
+    return "Player One wins overall, click on button to reset game.";
+  } else if (props.p1RoundsWon < props.p2RoundsWon) {
+    return "Player Two wins overall, click on button to reset game.";
+  } else if (props.p1RoundsWon === props.p2RoundsWon) {
+    return "This round ended with a draw, click on button to reset game.";
+  }
+}
```

Comment:
An alternative way of writing this for better readability. Overall I find this component possibly a bit redundant and we could maybe leave it as is in the parent component.

```suggestion
return (
    <>
        {props.p1RoundsWon > props.p2RoundsWon && "Player One wins overall, click on button to reset game."}
        {props.p1RoundsWon < props.p2RoundsWon && "Player Two wins overall, click on button to reset game."}
        {props.p1RoundsWon === props.p2RoundsWon && "This round ended with a draw, click on button to reset game."}
    </>
)
```